app/static/js/userProfile.js

```diff
@@ -61,6 +62,23 @@ $(document).ready(function(){
 
   });
 
+  $("#addNoteButton").click(function() {
+    $("#noteModal").modal("toggle");
+  });
+
+  $(".deleteNoteButton").click(function() {
+    let username = $(this).data('username')
+    let noteid = $(this).data('noteid')
+    $.ajax({
+      method: "POST",
+      url:  "/" + username + "/deleteNote",
+      data: {"id": noteid},
+      success: function(response) {
+        location.reload();
```

Comment:
This will reload the page and take the user off of the notes section. We want the user to stay in the notes section after the reload.

Reply:
fixed

Reply:
Fixed